theseus_gui/src-tauri/tauri.conf.json

```diff
@@ -100,8 +100,8 @@
         "resizable": true,
         "title": "Modrinth App",
         "width": 1280,
-        "minHeight": 700,
-        "minWidth": 1100,
+        "minHeight": 500,
+        "minWidth": 900,
```

Comment:
why was this decreased? does the launcher UI still look good at this size? i assume things (like onboarding) will probably be broken

Reply:
#669